sdcm/utils/common.py

```diff
@@ -1703,6 +1703,27 @@ def remove_files(path):
         LOGGER.info("Remove temporary data manually: \"%s\"", path)
 
 
+def create_remote_storage_dir(node, path='') -> Optional[str, None]:
+    node_remote_dir = '/tmp'
```

Comment:
don't use `/tmp` as it is cleaned on every boot, and we might loose the file we are trying to save

Reply:
no problem with that. even better.

Reply:
is it on the DB node? at which moment this is being sync'ed to the Runner/Builder?